shared-kcp/sharedkcp.kubeconfig

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+clusters:
+- cluster:
+    insecure-skip-tls-verify: true
+    server: https://api.ci-ln-01qrpgt-72292.origin-ci-int-gce.dev.rhcloud.com:6443
+  name: api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+- cluster:
+    server: https://kcp-stable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/root:rh-sso-15826143
+  name: kcp-stable
+- cluster:
+    server: https://kcp-unstable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/root:rh-sso-15826143
+  name: kcp-unstable
+- cluster:
+    server: https://kcp-stable.apps.kcp-internal-1.52p3.p1.openshiftapps.com/clusters/rh-sso-15826143:plsvc
+  name: workspace.kcp.dev/personal/plsvc
+contexts:
+- context:
+    cluster: api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+    namespace: default
+    user: kube:admin/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+  name: default/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443/kube:admin
+- context:
+    cluster: kcp-stable
+    namespace: default
+    user: oidc
+  name: kcp-stable
+- context:
+    cluster: kcp-unstable
+    namespace: default
+    user: oidc
+  name: kcp-unstable
+- context:
+    cluster: workspace.kcp.dev/personal/plsvc
+    user: oidc
+  name: workspace.kcp.dev/personal/plsvc
+current-context: workspace.kcp.dev/personal/plsvc
+kind: Config
+preferences: {}
+users:
+- name: kube:admin/api-ci-ln-01qrpgt-72292-origin-ci-int-gce-dev-rhcloud-com:6443
+  user:
+    token: sha256~5k1bZMZzeFomHBsugloXbBSPq-5VxOPABUmfmSoA8cc
```

Comment:
I'm not sure it's a great idea to commit this information. Please confirm with someone that this is not a security risk.
Also, the token is likely to have a TTL, which means that we'll have to update the file regularly and/or when we run the command a new token will be generated so the file will be shown as modified by git, will end up in PRs, and has a high risk of generating PR conflicts.

I would prefer if that info was removed. if the file was renamed `sharedkcp.template`, and it was copied by the script as `sharedkcp.kubconfig` with the latter file added to `.gitignore`.  If you do that, you might as well set the ORG_ID during the copy, that way if someone needs to reuse that script they don't need to mess with this file (my understanding is that you think these scripts might be useful to others).

Reply:
Yes I agree. User/cluster details is committed to Git by mistake. 

This script assumes that the sharedkcp.kubeconfig is already setup from the user. So the user needs to add their org ID and run kubectl api-resources (in order to login through their username). And also create their workspace. Creating a workspace will alter the contents of sharedkcp.kubeconfig (adds workspace details and changes current-context).

I have modified README to clearly reflect what this script does.